docs/deployment/dns.md

```diff
@@ -0,0 +1,77 @@
+---
+title: DNS Configuration
+type: docs
+weight: 2
+---
+
+UDS Core deploys two Gateways by default - a Tenant Gateway for end-user applications and an Admin Gateway for administrative applications. You can read more about Istio configuration in UDS Core [here](https://uds.defenseunicorns.com/core/configuration/istio/ingress/). This section covers how to configure DNS for these Gateways.
+
+### Domain Configuration
+Each Gateway is associated to a wildcard DNS entry that is derived from the `DOMAIN` [variable](https://github.com/defenseunicorns/uds-core/blob/e624d73f79bd6739b6808fbdbf5ca75ebb7c1d3c/src/istio/zarf.yaml#L8) in the UDS Core Istio package. When deploying UDS Core, you can expect two Gateways to be created that match the following domain names:
+- *.<DOMAIN> / Tenant Gateway
+- *.admin.<DOMAIN> / Admin Gateway
```

Comment:
![Screenshot 2024-09-19 at 8 13 12 AM](https://github.com/user-attachments/assets/07890787-9bd6-4401-a022-b8d528e21a8d)

These get clobbered on the docs site preview - could either escape the `<` and `>` characters, or just wrap in code backticks.

Reply:
Oops! how did you render this out?

Reply:
Using the [docs repo](https://github.com/defenseunicorns/uds-docs), if you have `hugo` installed locally you can just run `hugo serve`. You will want to clone or copy core into the `./repo-docs` folder first (I use a symlink locally for this), see the script we use to manage this: https://github.com/defenseunicorns/uds-docs/blob/main/scripts/integrated-docs.sh#L22

Reply:
Would love to better integrate/task-ify this flow: https://github.com/defenseunicorns/uds-core/issues/712

Reply:
Updated:
![image](https://github.com/user-attachments/assets/0862e359-0b3b-4624-ba97-267264a72d05)